xwiki-commons-core/xwiki-commons-context/src/main/java/org/xwiki/context/internal/concurrent/AbstractContextStore.java

```diff
@@ -39,6 +39,12 @@
     @FunctionalInterface
     protected interface SubContextStore
     {
+        /**
+         * Put in the context the value associated with the provided key prefix and suffix.
+         *
+         * @param key the main key (key prefix) of the value to save
+         * @param subkey the sub key (key suffix) of the value to save
+         */
```

Comment:
@tmortagne doesn't feel related to this PR? mistake on your side?

Reply:
Not a mistake, if you look at previous commits you will notice that I just made the javadoc introduced by @
@nmancus1 a bit more accurate ;)

Reply:
Right, ok.